packages/next-codemod/bin/transform.ts

```diff
@@ -41,22 +45,28 @@ export async function runTransform(
   }
 
   if (!path) {
-    const res = await prompts({
-      type: 'text',
-      name: 'path',
-      message: 'On which files or directory should the codemods be applied?',
-      initial: '.',
-    })
+    const res = await prompts(
+      {
+        type: 'text',
+        name: 'path',
+        message: 'On which files or directory should the codemods be applied?',
+        initial: '.',
+      },
+      { onCancel }
```

Comment:
wonder if we should create a `cancellablePrompt` wrapper? This seems fine for now